### PR TITLE
add support for --extra-modules

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review

File: easybuild/framework/easyblock.py [hunk 1]
```diff
@@ -1560,6 +1560,11 @@ def prepare_step(self, start_dir=True):
             # version is expected to be something that makes sense
             env.setvar(get_software_version_env_var_name(name), version)
 
+        extra_modules = build_option('extra_modules')
+        if extra_modules:
+            self.log.info("Loading extra modules: %s", extra_modules)
+            self.modules_tool.load(extra_modules)
```
Comment (wpoely86) [inline easybuild/framework/easyblock.py]: remind me, is this before or after loading the toolchain and deps?

Comment (boegel) [inline easybuild/framework/easyblock.py]: after, I use it for XALT, for example (where order matters a lot)

Comment (wpoely86) [inline easybuild/framework/easyblock.py]: perfect. We can use it too for `ccache` than.

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**